app/src/main/java/org/simple/clinic/summary/PatientSummaryViewRenderer.kt

```diff
@@ -3,10 +3,13 @@ package org.simple.clinic.summary
 import org.simple.clinic.mobius.ViewRenderer
 
 class PatientSummaryViewRenderer(
-    private val ui: PatientSummaryScreenUi
+    private val ui: PatientSummaryScreenUi,
+    private val modelUpdateCallback: PatientSummaryModelUpdateCallback?
```

Comment:
Is this nullable only because we don't want to pass it in during the tests? In this case, it should not be nullable and we should have a `mock` or a no-op used in tests.

Reply:
No, the `modelUpdateCallback` used in the `PatientSummaryScreen` is nullable. So that's why I had to make it nullable here as well. Should I instead make this non-null and have a use lambda and safe call in the screen?

Reply:
Made it non-null and went with no-op approach in tests.